app/assets/javascripts/discourse/lib/translation-plugin.js

```diff
@@ -47,7 +47,7 @@ class TranslationPlugin extends Plugin {
     this.inputPaths.forEach((path) => {
       let file = path + "/" + this.inputFile;
       let yaml = fs.readFileSync(file, { encoding: "UTF-8" });
-      let loaded = Yaml.load(yaml);
+      let loaded = Yaml.load(yaml, { json: true });
```

Comment:
Can't find the docs on that option, what does it do? 😃

Reply:
Basically waht my description says :)

> json (default: false) - compatibility with JSON.parse behaviour. If true, then duplicate keys in a mapping will override values rather than throwing an error.

Reply:
Also, can we use `safe_load` here? To be… safe 😛

Reply:
lmao, nevermind both comments, I've been looking at ruby docs 😂